oebue.xpr

```diff
@@ -332,7 +332,7 @@
     </meta>
     <projectTree name="oebue.xpr">
         <folder path="."/>
-        <file name="http://Herbst:9.1.149;346;96;428;261;281;89;350;288;138;243;149;173;210;218;302;282;324;269;282;@narragonien-latina.informatik.uni-wuerzburg.de/exist/webdav/db/projects/narragonia-latina/data/Kommentar/B_D579_Kommentar.xml"/>
-        <file name="http://Herbst:9.1.173;214;176;151;203;215;121;172;127;153;89;165;165;157;155;148;298;137;423;221;@narragonien-latina.informatik.uni-wuerzburg.de/exist/webdav/db/projects/narragonia-latina/data/Kommentar/yanniks_test.xml"/>
+        <file name="http://Herbst:9.1.97;300;140;432;266;283;139;303;288;93;248;193;218;210;218;251;331;327;269;322;@narragonien-latina.informatik.uni-wuerzburg.de/exist/webdav/db/projects/narragonia-latina/data/Kommentar/B_D579_Kommentar.xml"/>
```

Comment:
Solche Änderungen bitte nicht mit reinnehmen.

Reply:
Welche Änderungen meinst du denn genau? Und wie nehme ich diese in Zukunft nicht mehr mit rein? Soll ich die Änderungen entfernen und wenn ja, wie mache ich das?

Reply:
Beim git add nicht die obue mit reinnehmen :+1:

Reply:
Funktioniert das dann durch das gitignore oder muss ich dafür noch extra etwas schreiben?